kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -125,41 +126,42 @@ private void updateAllPartitions(String topic, Map<TopicPartition, OffsetAndMeta
   }
 
   private class TableWriter implements Callable<Void> {
-    private final TableId table;
+    private final PartitionedTableId partitionedTableId;
     private final List<RowToInsert> rows;
     private final Map<TopicPartition, OffsetAndMetadata> offsets;
     private final String topic;
     private final Set<Schema> schemas;
     private final BatchWriter<RowToInsert> batchWriter;
 
-    public TableWriter(TableId table,
+    public TableWriter(PartitionedTableId partitionedTableId,
                        List<RowToInsert> rows,
                        Map<TopicPartition, OffsetAndMetadata> offsets,
                        String topic,
                        Set<Schema> schemas) {
-      this.table = table;
+      this.partitionedTableId = partitionedTableId;
       this.rows = rows;
       this.offsets = offsets;
       this.topic = topic;
       this.schemas = schemas;
-      this.batchWriter = batchWriterManager.getBatchWriter(table);
+      this.batchWriter = batchWriterManager.getBatchWriter(partitionedTableId.getBaseTableId());
     }
 
     @Override
     public Void call() throws InterruptedException {
-      batchWriter.writeAll(table, rows, topic, schemas);
-      updateAllPartitions(tableIdsToTopics.get(table), offsets);
+      batchWriter.writeAll(partitionedTableId.getFullTableId(), rows, topic, schemas);
+      updateAllPartitions(baseTableIdsToTopics.get(partitionedTableId.getBaseTableId()), offsets);
       return null;
     }
   }
 
   /**
-   * A class for keeping track of the BatchWriters for each table.
+   * A class for keeping track of the BatchWriters for each partitionedTableId.
```

Comment:
I don't actually see any usage of partitionedTableId in this class. Is this comment accurate?